framework/doc/service_hooks.rst

```diff
@@ -14,9 +14,9 @@ application bundles.
 
 Some example use cases for service hooks include:
 
-- Proxying of existing services by hiding the original service and 
-registering a *proxy service* with the same properties 
-- Providing services *on demand* based on registered service listeners from external bundles
+   - Proxying of existing services by hiding the original service and 
+     registering a *proxy service* with the same properties 
+   - Providing services *on demand* based on registered service listeners from external bundles
 
```

Comment:
When you mentioned "missing indentation", could you possibly clarify what you mean by that? I thought you meant add an indentation, but I think I am misunderstanding you.